src/tests/DevIntegration.test.ts

```diff
@@ -426,3 +427,128 @@ describe("INTEGRATION dev environment tests", () => {
         });
     });
 });
+
+describe("INTEGRATION dev environment deterministic tests", () => {
```

Comment:
A test that encrypt and search terms match up would be nice, and one showing encrypt matches the most recent secret for search on one in rotation.